roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -116,15 +117,104 @@ class TransitionSpec extends FlatSpec with Matchers {
         .set(_ >> 'ctxt >> 'ctxt)(testState.ctxt)
 
     val codevec = Seq(OpAlloc(2),
-                      OpImmediateLitToArg(v = 1, a = 0),
-                      OpImmediateLitToArg(v = 2, a = 1),
-                      OpXferGlobalToReg(r = 1, g = 668),
-                      OpXmit(u = false, n = true, 2))
+                      OpImmediateLitToArg(value = 1, arg = 0),
+                      OpImmediateLitToArg(value = 2, arg = 1),
+                      OpXferGlobalToReg(reg = 1, global = 668),
+                      OpXmit(unwind = false, next = true, 2))
 
     val end = VirtualMachine.executeSeq(codevec, start)
     end.ctxt.ctxt.rslt shouldBe Fixnum(3)
   }
 
+  "Executing bytecode from expression \"(+ 1 (+ 2 3))\"" should "result in Fixnum(6)" in {
+
+    /**
+      *
+      * litvec:
+      *  0:   {RequestExpr}
+      * codevec:
+      *  0:   alloc 2
+      *  1:   lit 1,arg[0]
+      *  2:   xfer global[+],trgt
+      *  3:   outstanding 12,1
```

Comment:
This is not adjusted which is fine but you adjusted the position numbers.
Let's keep the position numbers like they are when you run `(show-code (compile '(...)))`.
Also let's add a comment to the test suite which explains the difference.

Reply:
Ok, sounds reasonable.